Form/Type/InfoCollectionType.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Netgen\Bundle\EzFormsBundle\Form\Type;
+
+use eZ\Publish\API\Repository\Values\Content\ContentCreateStruct;
+use Symfony\Component\Form\FormBuilderInterface;
+use Netgen\Bundle\EzFormsBundle\Form\DataWrapper;
+use RuntimeException;
+
+class InfoCollectionType extends AbstractContentType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return "ezforms_info_collection";
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm( FormBuilderInterface $builder, array $options )
+    {
+        /** @var $dataWrapper \Netgen\Bundle\EzFormsBundle\Form\DataWrapper */
+        $dataWrapper = $options["data"];
+
+        if ( !$dataWrapper instanceof DataWrapper )
+        {
+            throw new RuntimeException(
+                "Data must be an instance of Netgen\\EzFormsBundle\\Form\\DataWrapper"
+            );
+        }
+
+        $contentCreateStruct = $dataWrapper->payload;
```

Comment:
Content create struct should not be needed as a payload, only content type. Creating collected info forms *does not* create content, so why should you need content create struct?